Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -218,6 +218,14 @@ public final class SecureEnclaveValet: NSObject {
         try migrateObjects(matching: valet.baseKeychainQuery, removeOnCompletion: removeOnCompletion)
     }
 
+    // MARK: Deprecation stubs
```

Comment:
```suggestion
    // MARK: Renamed methods
```

Reply:
nit: `Renamed Methods` to match capitalization of other marks

Reply:
updated everywhere